package.json

```diff
@@ -29,6 +29,9 @@
 		"@wordpress/wxr": "file:packages/wxr",
 		"cheerio": "1.0.0-rc.5",
 		"dayjs": "1.10.4",
+		"file-saver": "2.0.5",
```

Comment:
I think this was accidentally included in a rebase, perhaps. We don't use that package any more.